docs/sources/features/datasources/elasticsearch.md

```diff
@@ -162,6 +162,24 @@ Time | The name of the time field, needs to be date field.
 Text | Event description field.
 Tags | Optional field name to use for event tags (can be an array or a CSV string).
 
+## Querying Logs (BETA)
+
+Querying and displaying log data from Elasticsearch is available via [Explore](/features/explore).
+
+Select the Elasticsearch data source, change to Logs using the Metrics/Logs switcher,
+
+![](/img/docs/elasticsearch/explore_elasticsearch_logs_switch.png)
+
+and then enter your query using [Lucene](https://lucene.apache.org/core/2_9_4/queryparsersyntax.html) syntax into the query field
```

Comment:
> and then enter your query using [Lucene](https://lucene.apache.org/core/2_9_4/queryparsersyntax.html) syntax into the query field

and then optionally enter a lucene query into the query field to filter the log messages.